src/index.html

```diff
@@ -4,9 +4,46 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title><%= htmlWebpackPlugin.options.title %></title>
+  <script src="https://kit.fontawesome.com/4e57741acf.js" crossorigin="anonymous"></script>
+  <title>To-Do List</title>
 </head>
 <body>
-  <h1>Is my templete are working?</h1>
+  <main>
+    <div class="container">
+      <div class="item">
+        <div class="heading">Today's To Do</div>
+        <i class="fas fa-sync sync-btn"></i>
+      </div>
+      <div class="item">
+        <input class="task-adder-input" type="text" placeholder="Add to your list...">
+        <i class="fas fa-arrow-circle-left add-task-btn"></i>
+      </div>
+      <div class="item">
+        <div class="tasks-container">
+          <!-- <div class="task">
+            <div class="task-cont">
+              <input type="checkbox" name="task" id="task">
+              <input type="text" value="Play Guitar" class="task-value">
+            </div>
+            <div class="opp-icon">
+              <i class="fa fa-ellipsis-v dots-icon"></i>
+            </div>
+          </div>
+          <div class="task">
+            <div class="task-cont">
+              <input type="checkbox" name="task" id="task">
+              <input type="text" value="Play Guitar" class="task-value">
+            </div>
+            <div class="opp-icon">
+              <i class="fa fa-ellipsis-v dots-icon"></i>
+            </div>
+          </div> -->
```

Comment:
- [ ] You did great😃on the project, but I see that you left some unnecessary comments in the codebase that make your codebase ugly 😊, so it will be better if you remove these unnecessary comments to make the codebase clean and follow the best practice like the professionals 💯🚀